### ISSUE TITLE
Cannot use GitHub branch to create projects on Windows

### ISSUE THREAD

**Issue by TheSpydog** [HaxePunk/HaxePunk#587]
- **HaxePunk version:** 4.0.1 (dev branch)
- **Target(s):** N/A
- **Haxe version:** 3.4.7

Attempting to create a new Haxepunk project (`haxelib run haxepunk new <name>`) from the GitHub dev branch on Windows causes the following error:
```
'make' is not recognized as an internal or external command, operable program or batch file.
Called from ? line 1
Called from CLI.hx line 148
Called from CLI.hx line 10
Called from CLI.hx line 29
Called from CLI.hx line 42
Called from Project.hx line 85
Called from Project.hx line 97
Called from C:\HaxeToolkit\haxe\std/neko/_std/sys/io/File.hx line 50
Uncaught exception - [file_open,C:/HaxeToolkit/haxe/lib/HaxePunk/git/template.zip]
```

Is there any way to build projects using something other than a makefile, since Windows doesn't natively support `make`?

**Comment by bendmorris** [comment]
We do need a solution for this.

As a workaround, this path is only hit when template.zip doesn't exist, so you can create it yourself the same way the Makefile would:

`cd template && zip -rqX ../template.zip . -x *.DS_Store*`

(That's bash, you'd need something like Git Bash or the equivalent cmd on Windows, YMMV)

**Comment by JamesFalero** [comment]
For those who have 7zip stand alone console installed and in their PATH environment variable.

1. Install git for windows (I'm using Git-2.16.2-64-bit for reference)

2. Go to console and type "haxelib path HaxePunk" (without quotes) to figure out where HaxePunk is installed

3. Open the file called 'make' in a text editor of your choice.

4. Go to line thirty, or the line that says "cd template && zip -rqX ../template.zip . -x *.DS_Store*" 

5. Change line thirty to "cd template && zip a ../template.zip . x *.DS_Store*" (again, no quotes)

Finally, run haxelib "run haxepunk new YourProjectName" on the command line, as per usual.

It's a little extra step but it worked for me. Let me know how it works for any of you.

**Comment by Ghouly-The-Ghost** [comment]
I will write a small fix for this.

**Comment by Ghouly-The-Ghost** [comment]
A fix was merged for this issue. Unzipping is now done through Haxe using Sys api. We should be able to close this.

**Comment by matrefeytontias** [comment]
Fixed as of #612
